.github/workflows/build.yml

```
@@ -102,6 +102,8 @@ jobs:
           - {goos: "linux", goarch: "arm64"}
           - {goos: "linux", goarch: "386"}
           - {goos: "linux", goarch: "amd64"}
+          - {goos: "darwin", goarch: "arm"}
```

This was an error in my original consul PR. It should be:
```suggestion
          - {goos: "darwin", goarch: "amd64"}
```